justfile

```diff
@@ -47,27 +57,29 @@ default:
 check: && (check-crate _d1_pkg) (check-crate _espbuddy_pkg) (check-crate _x86_bootloader_pkg)
     {{ _cargo }} check \
         --lib --bins --examples --tests --benches \
-        {{ _fmt }}
+        {{ _fmt_check_doc }}
 
 # check a crate.
 check-crate crate:
     {{ _cargo }} check \
         --lib --bins --examples --tests --benches --all-features \
         --package {{ crate }} \
-        {{ _fmt }}
+        {{ _fmt_check_doc }}
 
 # run Clippy checks for all crates, across workspaces.
+# NOTE: -Dwarnings is added by _fmt because reasons
```

Comment:
sadly, the way Just handles multiline comments is...a bit unfortunate. the line of comment immediately above the recipe is used to generate the description in `just --list`. what that means is that on `main`, the `just --list` output looks like this:
```console
$ just --list
Available recipes:
    all-docs *FLAGS               # build all RustDoc documentation
    build-c3 board                # build a MnemOS binary for the ESP32-C3
    build-d1 board='mq-pro'       # build a Mnemos binary for the Allwinner D1
    build-x86 *args=''            # build a bootable x86_64 disk image, using rust-osdev/bootloader.
    check                         # check all crates, across workspaces
    check-crate crate             # check a crate.
    clippy                        # run Clippy checks for all crates, across workspaces.
    clippy-crate crate            # run clippy checks for a crate.
    crowtty *FLAGS                # run crowtty (a host serial multiplexer, log viewer, and pseudo-keyboard)
    default                       # default recipe to display help information
    docs *FLAGS                   # run RustDoc
    flash-c3 board *espflash-args # flash an ESP32-C3 with the MnemOS WiFi Buddy firmware
    flash-d1 board='mq-pro'       # flash an Allwinner D1 using xfel
    fmt                           # run rustfmt for all crates, across workspaces
    melpomene *FLAGS              # run the Melpomene simulator
    melpo *FLAGS                  # alias for `melpomene`
    run-x86 *args=''              # run an x86_64 MnemOS image in QEMU
    test                          # test all packages, across workspaces
```
but on this branch, it looks like this instead:
```console
$ just --list
Available recipes:
    all-docs *FLAGS               # build all RustDoc documentation
    build-c3 board                # build a MnemOS binary for the ESP32-C3
    build-d1 board='mq-pro'       # build a Mnemos binary for the Allwinner D1
    build-x86 *args=''            # build a bootable x86_64 disk image, using rust-osdev/bootloader.
    check                         # check all crates, across workspaces
    check-crate crate             # check a crate.
    clippy                        # NOTE: -Dwarnings is added by _fmt because reasons
    clippy-crate crate            # NOTE: -Dwarnings is added by _fmt because reasons
    crowtty *FLAGS                # run crowtty (a host serial multiplexer, log viewer, and pseudo-keyboard)
    default                       # default recipe to display help information
    docs *FLAGS                   # run RustDoc
    flash-c3 board *espflash-args # flash an ESP32-C3 with the MnemOS WiFi Buddy firmware
    flash-d1 board='mq-pro'       # flash an Allwinner D1 using xfel
    fmt                           # run rustfmt for all crates, across workspaces
    melpomene *FLAGS              # run the Melpomene simulator
    melpo *FLAGS                  # alias for `melpomene`
    run-x86 *args=''              # run an x86_64 MnemOS image in QEMU
    test                          # test all packages, across workspaces
```
so, the `# NOTE: ...` replaces the actual description of the recipe.

Let's fix that, either by moving the `NOTE: ...` comment _above_ the recipe description, or removing it?